src/cms/page_xliff_converter.py

```diff
@@ -491,11 +491,20 @@ def generate_xliff_import_diff(self, xliff_paths):
                     trans_fields = converter.xliff_to_translation_data()
                     if trans_fields is None:
                         continue
-                    diffs.append(
-                        self.generate_translation_diff(
-                            trans_fields, os.path.basename(xliff_path)
-                        )
+                    diff = self.generate_translation_diff(
+                        trans_fields, os.path.basename(xliff_path)
                     )
+                    if diff is None:
+                        diffs.append(
+                            {
+                                "error": True,
```

Comment:
I'm not a 100% happy with this solution, as this is an "abuse" of the structure of the `diffs` list/dict. However, I did like the alternative of returning 2 variables or even restructuring everything even less.

Reply:
We could rename the variable `diff` to `result` to indicate that it can hold also non-diff-objects, but in general I think this is totally fine.